content/games.html

```diff
@@ -166,6 +175,11 @@
 <noscript><div class="noscript">Please enable JavaScript to view the server list.</div></noscript>
 <div id="servercontainer" style="display: none">
 <p><span id="activePlayers">0</span> players are online in <span id="activeServers">0</span> advertised servers.</p>
+<form name="show">
+  <b>Show:</b>
+  <input type="checkbox" name="started" onchange="updateGameList()" checked=true> already started
```

Comment:
Not sure if we want it, but you could do e.g. `<label for="status_started" class="gamePlaying">Playing</label>` to colour the labels too.